src/components/Contact/index.js

```diff
@@ -1,14 +1,75 @@
 import React, { Component } from 'react';
 import './Contact.css';
 
+
 class Contact extends Component {
     render() {
         return (
-            <div className="page">
+             <div className="page">
                 <div className="pageHeader">Contact Us</div>
-                <div>
-                    Contact us here (links here)!!!
+                <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css" />
+                <script src="https://kit.fontawesome.com/index.js"></script>
+
+             <div className="sentence">
+                <br></br>
+                If you would like to contact us, please send an email to&nbsp;
+                    <a href="mailto:stuyvesantflow@gmail.com?subject=test%20email"><br></br>stuyvesantflow@gmail.com</a>
+                    <br></br>
+
                 </div>
+                <br></br>
+                <div className="sentence">
+                  <br></br>
+                    Click the icons below to check us out on our social media!
+                </div>
+                <br></br>
+
+                <a href="https://www.facebook.com/StuyFl0w" class="fab fa-facebook-square" style={{fontSize: '40px'}}></a>&nbsp;&nbsp;&nbsp;
+                <a href="https://www.youtube.com/channel/UCsWvNNE26HT40Tk7O9afkSA" class="fab fa-youtube" style={{fontSize: '38px'}}></a>&nbsp;&nbsp;&nbsp;
+                <a href="https://www.instagram.com/stuyflow/" class="fab fa-instagram" style={{fontSize: '40px'}}></a>&nbsp;&nbsp;&nbsp;
+                <a href="https://www.tiktok.com/@stuyflow?lang=en" class="fab fa-tiktok"style={{fontSize: '40px'}}></a>
+
```

Comment:
A couple things:
- Change `class` to `className` because we are using React
- Since each <a> tag has its own custom `fa-<insert social media>` class, then put the font size in the CSS file
- Please do not use &nbsp; Although it does make it easier to space stuff, when we transfer to mobile, it might not have the same spacing. We already have the `fab` class that we can use to add margins to each of the <a> tags.
- This is optional but maybe find some way to download the icons. It seems kind of overkill to import an entire library just to get 4 icons. When you hard refresh, you can see a bit of delay.

Reply:
Another thing: add `target='_blank'` as an attribute to each <a> tag. We want to open the social media on a different tab so the user can stay go back to the page if they want to.